components/MissionSection.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-  <section class="mt-20 w-full bg-[url('/banner-homepage.png')] bg-cover bg-center flex justify-center">
-    <div class="inline-grid gap-8 max-w-4xl p-20 pl-52 self-end">
+  <section class="mt-20 w-full bg-[url('/banner-homepage.png')] bg-cover bg-left flex justify-center">
```

Comment:
prob should but bg-top for mobile here, since the text color doesn't have enough contrast imho